### PR TITLE
*: fix ci build by checking out a previous version of `pd`.

### PR DESCRIPTION
**Pull request by coocood**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -49,6 +49,8 @@ godep:
 	go get github.com/pingcap/go-hbase
 	go get github.com/pingcap/go-themis
 	go get github.com/pingcap/tso/client
+	go get github.com/pingcap/pd/pd-client
```
Comment (siddontang) [inline Makefile]: `go get -d` can solve this ?

**Comment by zimulala** [comment]
LGTM

File: Makefile (hunk 1 above)
Comment (coocood) [inline Makefile]: Not for now.

**Comment by siddontang** [comment]
LGTM

But I suggest you should change it after pd is ready.

**Pull request merged**